[CCI] update contributing.md (#3794)
opensearch-project/OpenSearch-Dashboards: CONTRIBUTING.md

Once you've opened an issue, check out our [Developer Guide](./DEVELOPER_GUIDE.md) for instructions on how to get started.

> _Note_: Since you are a beginner, we advise you to start with simple issues.

Reviewer: Not all readers of this document are beginners. We should rephrase to be something like, "The easiest way to get started is with simple issues".

> For this there is a special label "<font color="SlateBlue">**Good first issue**</font>"

Reviewer: I think GitHub will render a label link as the link itself.
```suggestion
For this there is a special label https://github.com/opensearch-project/OpenSearch-Dashboards/labels/good%20first%20issue
```

> Warning! If you have selected an issue and want to get started, check if the issue is free. If for it has already been assigned <br> and there is no pull request for a long time, ask that user using @ (e.g @nickname) if he/she is willing to concede to you.

Reviewer: ```suggestion
Warning! Before getting started, check if the issue is free. If it has already been assigned for a long time and there isn't a pull request yet, ask the assigned user using @ (e.g @nickname) if you can work on it instead.
```

> If the issue has recently been created and there are no appointees, then feel free to write in a comment that you are ready to take on the issue.

Reviewer: ```suggestion
If the issue has no assignees, then feel free to write in a comment that you are ready to take on the issue.
```

Feel free to create an issue if you have not been able to work out the installation, or a bug within the project. Also feel free to ask on the issue page if you are solving a problem but can't figure out the structure of the project.

> 

Reviewer: This whole topic should be moved to the "Contributing Code" section. The idea is that opening and participating in issues is one type of contribution, separate from providing code.

    sign-off) is maintained indefinitely and may be redistributed
    consistent with this project or the open source license(s)
    involved.
```

We require that every contribution to OpenSearch is signed with a Developer Certificate of Origin. Additionally, please use your real name. We do not accept anonymous contributors nor those utilizing pseudonyms.

Each commit must include a DCO which looks like this

```
Signed-off-by: Jane Smith <jane.smith@email.com>
```

You may type this line on your own when writing your commit messages. However, if your user.name and user.email are set in your git configs, you can use `-s` or `--signoff` to add the `Signed-off-by` line to the end of the commit message.

> Fill in your personal details as well as an email that is public on github:

Reviewer: ```suggestion
Configure `git` with your name and an email that is public on github:
```

```
git config --global user.name "FIRST_NAME LAST_NAME"
git config --global user.email "MY_NAME@example.com"
```

_Note_: If you want to set up this data only for a specific repository then do it without the `--global` parameter:

```
git config user.name "FIRST_NAME LAST_NAME"
git config user.email "MY_NAME@example.com"
```

> ## First pull request

Reviewer: This section should be a separate document. Because it's generic, I'd suggest proposing a new document in https://github.com/opensearch-project/.github
